test/framework/cluster.go

```diff
@@ -583,29 +586,65 @@ func (e *ClusterE2ETest) WithClusterConfig(fillers ...api.ClusterConfigFiller) *
 	return e
 }
 
-func (e *ClusterE2ETest) ImportImages(opts ...CommandOpt) {
-	importImagesArgs := []string{"import-images", "-f", e.ClusterConfigLocation}
-	e.RunEKSA(importImagesArgs, opts...)
-}
-
+// DownloadArtifacts runs the EKS-A `download artifacts` command with appropriate args.
 func (e *ClusterE2ETest) DownloadArtifacts(opts ...CommandOpt) {
 	downloadArtifactsArgs := []string{"download", "artifacts", "-f", e.ClusterConfigLocation}
+	if getBundlesOverride() == "true" {
+		downloadArtifactsArgs = append(downloadArtifactsArgs, "--bundles-override", defaultBundleReleaseManifestFile)
+	}
 	e.RunEKSA(downloadArtifactsArgs, opts...)
-	if _, err := os.Stat("eks-anywhere-downloads.tar.gz"); err != nil {
+	if _, err := os.Stat(defaultDownloadArtifactsOutputLocation); err != nil {
 		e.T.Fatal(err)
 	} else {
-		e.T.Log("Downloaded artifacts saved at eks-anywhere-downloads.tar.gz")
+		e.T.Logf("Downloaded artifacts tarball saved at %s", defaultDownloadArtifactsOutputLocation)
 	}
 }
 
-// ExtractDownloadedArtifacts extract the downloaded artifacts.
+// ExtractDownloadedArtifacts extracts the downloaded artifacts.
 func (e *ClusterE2ETest) ExtractDownloadedArtifacts(opts ...CommandOpt) {
-	if _, err := os.Stat("eks-anywhere-downloads.tar.gz"); err != nil {
+	e.T.Log("Extracting downloaded artifacts")
+	e.Run("tar", "-xf", defaultDownloadArtifactsOutputLocation)
+}
+
+// DownloadImages runs the EKS-A `download images` command with appropriate args.
+func (e *ClusterE2ETest) DownloadImages(opts ...CommandOpt) {
+	downloadImagesArgs := []string{"download", "images", "-o", defaultDownloadImagesOutputLocation}
+	if getBundlesOverride() == "true" {
+		var bundleManifestLocation string
+		if _, err := os.Stat(defaultDownloadArtifactsOutputLocation); err == nil {
+			bundleManifestLocation = "eks-anywhere-downloads/bundle-release.yaml"
```

Comment:
the 2 lines should execute if getBundlesOverride() == "false"

Reply:
`getBundlesOverride()` method just reads the T_BUNDLES_OVERRIDE env var, but we are providing a choice of which local bundle manifest to use as override. For air-gapped, we will use the one from download-artifacts, but otherwise the one from `bin/local-bundle-release.yaml`

Reply:
fair enough, I'm not too much concernd about this, let's see how it goes